sql/rooms/up.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE Rooms (
     "capacity"      INTEGER NOT NULL,
     "school"        TEXT NOT NULL,
     "buildingId"    TEXT NOT NULL,
+    "facilities"    JSONB NOT NULL,
```

Comment:
The more I think about this the more I feel you should be making a field for each of the categories. The JSONB here is a minor convenience more than anything useful, and probably is just less efficient in some way.

Since the data is pretty structured anyway, there isn't really a need to use JSONB. Adding fields allows for some extra validation as well (e.g. create enum type for each facility category, enforce single type of floor/seating, etc.).

Reply:
Do we just want to add a field for each categories? In cases where a field can have multiple entries (eg Audio-visual), we would still need to store the data as a JSONB array. We can go with the full normalization approach and have a table for facilities, and a junction table between room and facilities. 

I went with JSONB since I imagine we can do the filtering and casting on the backend, and the database only stores the pure scraped data, but if we are structuring the data before inserting into the database and the backend can consume the data as-is, I think a junction table approach fits best.

Reply:
i am blind

Reply:
In `sql/buildings/up.sql`